src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,21 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    public static void main(String[] args) {
```

Comment:
Avoid using static methods for functionality that can be encapsulated within an object. Consider refactoring the main method to call instance methods on objects.